sensitivity-api/src/test/java/com/powsybl/sensitivity/mocks/AnotherSensitivityAnalysisProviderMock.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.sensitivity.mocks;
+
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.contingency.ContingenciesProvider;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.sensitivity.SensitivityAnalysisParameters;
+import com.powsybl.sensitivity.SensitivityAnalysisProvider;
+import com.powsybl.sensitivity.SensitivityAnalysisResults;
+import com.powsybl.sensitivity.SensitivityFactorsProvider;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public class AnotherSensitivityAnalysisProviderMock implements SensitivityAnalysisProvider {
```

Comment:
Do you forget the `@AutoService`?

Reply:
Yes !

Reply:
I actually deleted this class, it was not used in the tests anymore